diff --git a/secure_element/aidl/Android.bp b/secure_element/aidl/Android.bp
index 1145305..af7fa38 100644
--- a/secure_element/aidl/Android.bp
+++ b/secure_element/aidl/Android.bp
@@ -11,6 +11,7 @@ package {
 aidl_interface {
     name: "android.hardware.secure_element",
     vendor_available: true,
+    recovery_available: true,
     host_supported: true,
     srcs: ["android/hardware/secure_element/*.aidl"],
     stability: "vintf",
diff --git a/weaver/aidl/Android.bp b/weaver/aidl/Android.bp
index c494d29..d061555 100644
--- a/weaver/aidl/Android.bp
+++ b/weaver/aidl/Android.bp
@@ -11,6 +11,7 @@ package {
 aidl_interface {
     name: "android.hardware.weaver",
     vendor_available: true,
+    recovery_available: true,
     srcs: ["android/hardware/weaver/*.aidl"],
     stability: "vintf",
     backend: {
